README.md

```diff
@@ -53,7 +53,7 @@ md \dev\ros2_dotnet_ws\src
 cd \dev\ros2_dotnet_ws
 curl -sk https://raw.githubusercontent.com/esteve/ros2_dotnet/master/ros2_dotnet.repos -o ros2_dotnet.repos
 vcs import \dev\ros2_dotnet_ws\src < ros2_dotnet.repos
-src\ament\ament_tools\scripts\ament.py build
+colcon build --merge-install
```

Comment:
FYI the necessary fixes to `colcon-cmake` (for the -A flag) are *not* included in the latest colcon meta-release. May want to add a note that the user should ensure this subpackage is version 0.2.12 or higher.

Reply:
That's strange, I added `colcon` via `colcon-common-extensions` to the CI and it seems to have accepted the `-A` flag, do you think the instructions should have a call to `pip install colcon-cmake>=0.2.12`?